### PR TITLE
General Updates

### PR DESCRIPTION
**Pull request by Christian-Oleson**
I'm reading through the code to get a better sense of what is covered and where we need to add features to be in parity with Kubernetes' abilities. As I go through this, I'm finding some issues in the code and correcting where possible.

### CODE REVIEW

**Comment by Christian-Oleson** [comment]
I signed the CLA.

File: src/KubernetesClient/Enums/WatchEventType.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+﻿using System.Runtime.Serialization;
+
+namespace k8s.Enums
+{
+    /// <summary>Describes the type of a watch event.</summary>
+    public enum WatchEventType
```
Comment (tg123) [inline src/KubernetesClient/Enums/WatchEventType.cs:6]: why do you want to move ns?
this is a breaking change

File: src/KubernetesClient/generated/IKubernetes.Watch.cs [hunk 2]
```diff
@@ -7,7 +7,8 @@
 using System; 
 using System.Collections.Generic; 
 using System.Threading; 
-using System.Threading.Tasks; 
+using System.Threading.Tasks;
```
Comment (tg123) [inline src/KubernetesClient/generated/IKubernetes.Watch.cs:10]: generated code should be modified by code generator only

**Review by tg123 — COMMENTED** [review]
not quite get the idea of this PR?
seems nothing changed but some code format?

File: src/KubernetesClient/Enums/WatchEventType.cs (hunk 1 above)
Comment (Christian-Oleson) [reply src/KubernetesClient/Enums/WatchEventType.cs:6]: I can not move namespaces, but it seems (and perhaps I'm wrong here) that we should break up classes so they don't all live in the root of the source?

File: src/KubernetesClient/generated/IKubernetes.Watch.cs (hunk 2 above)
Comment (Christian-Oleson) [reply src/KubernetesClient/generated/IKubernetes.Watch.cs:10]: > not quite get the idea of this PR?
> seems nothing changed but some code format?
Yes, more or less. This is just an initial PR to take a look at the code, clean up some issues I found while reading along the way, and get a first PR into the repo. I wasn't aware that some of this was code generated, so I'll take a look further and go from there. 😄

Comment (Christian-Oleson) [reply src/KubernetesClient/generated/IKubernetes.Watch.cs:10]: Ah, I see now the error. This is generated code. Closing the pr...

**Pull request closed without merging**